### ISSUE TITLE
Remove EDR/XML Bulk Data Files with Release of ECMPS 2.0

### ISSUE THREAD

**Issue by tlageman-epa** [US-EPA-CAMD/PI-1#5497]
### Definition of Ready
- [ ] Does this meet the INVEST criteria?
- [ ] Does this have a clearly defined user?
- [ ] Does this have an estimate?
- [ ] Does this have acceptance criteria?
- [ ] Does this have sufficient context for the specific functionality being developed (design, business rules, etc.)
- [ ] Does this have a dev link been provided in the design?
- [ ] Is this traceable to a feature, epic, or PI?

## User Story
As a 
I want
So that

## Acceptance Criteria
Given
When
Then

### Does this Story require 508 Testing?
- [ ] Implementing a new authoring tool that will produce code / content for the application
- [ ] Implementing a 3rd party widget that will be part of the application
- [ ] Adding new / updating custom functionality 
- [ ] Adding new / updating a page template
- [ ] Adding new / replacing existing functionality (unless the replacement has already passed 508 testing)
- [ ] Full Site Review prior to a scheduled EPA 508 review
- [ ] Planning a video that will appear on the application
- [ ] Producing a video that will appear on the application
- [ ] Producing Training materials
- [ ] Producing documents

## Context
7/12/23 - To avoid creating JSON files with ECMPS 2.0 release and based on lack of users who use XMLs and EDRs, EPA has decided to remove the existing XML and EDR files from CAMPD's Bulk Data Files. A notice will be put up at the start of the new year (2024) on the Bulk Data Files page to warn users that files will be taken down with ECMPS 2.0 launch.  The official copy of the XML and EDRs is in the database so there is no issue deleting these. Additionally, XMLs would not be updated once ECMPS 2.0 is live and therefore would be archived like EDRs. The raw submittal data provided by the EDRs and XMLs will be available via ECMPS 2.0 and eventually, CAMPD will add in a Monitoring Level Emissions endpoint(s) (see ticket 5078) to provide users with easier access to the monitoring level emissions data that was in the XML/EDRs.

3/27/24(tlageman) - due to ECMPS 2.0 delay, I recommend putting up a notice on the CAMPD Bulk Data Files page 1 quarter before the planned removal. It may also be worthwhile (if not too much effort) to remove the files 1 month before from the UI/Download, but not delete them on the off chance there is a backlash of users asking for this data. You'd have to check with Mike, but if the file hosting storage cost is minimal you could leave these in an archival state but just make any users who are concerned about the data going away that they will no longer be updated/maintained.

## Testing
Test Case:
Tester:

### Definition of Done
- [ ] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [ ] Has the code been unit tested and peer reviewed?
- [ ] Have the functional tests been executed?
- [ ] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [ ] Is the user story ready to be deployed to the test/staging environments?

**Comment by alangmaid** [comment]
@tlageman-epa should this ticket be tied to 2.0 release and not v1.4?

**Comment by tlageman-epa** [comment]
@alangmaid yes and no. If campd 1.4 can launch before ecmps 2.0 **and** 1.4 includes the monitoring level emissions data work, then this ticket should be part of 1.4. If  the monitoring level work can't be done by then, then it can wait until ecmps 2.0 is released.

**Comment by alangmaid** [comment]
Added that this issue is blocked by 5078

**Comment by tlageman-epa** [comment]
@annalbrecht I realized i did already make a ticket for this. see context for details. Also to adam's question above, at this point yes, this would be part of CAMPD 2.0 and ECMPS 2.0 and is irrespective of any future planned monitoring level data we want to add into CAMPD. To be further clear. this issue is not blocked by 5078.
